agent/mapcall/src/main/resources/default-load-exclusions.txt

```diff
@@ -0,0 +1,27 @@
+// This file contains default package exclusions for the mapcall agent.
+
+// Don't transform Java classes that are not loaded by the boot loader, but
+// for which the transformation leads to failures.
+sun.reflect.
+com.sun.proxy.$Proxy.
+
+// Don't transform Randoop classes:
+randoop.
+mapcall.
```

Comment:
Since this relates to building the jar files, I won't try this until after everything is merged.